library/Core.cpp

```diff
@@ -2416,7 +2416,7 @@ int Core::Shutdown ( void )
     d->hotkeythread.join();
     d->iothread.join();
 
-    CoreSuspendClaimer suspend;
```

Comment:
This shouldn't be here at all. This is in `Core::Shutdown`, the core suspension mechanism has already been dismantled at this point and all DFHack threads. as well as the DF simulation thread, have been terminated. The suspender should be removed entirely since there's nothing to protect against here.

Reply:
Fixed in https://github.com/DFHack/dfhack/commit/ebac3772cec2f9150f562a50daac63e8dfa9634d